.github/workflows/ci.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         if: ${{ matrix.platform == 'ubuntu-latest' }}
         run: make test-e2e
 
+      - name: Run Example tests
```

Comment:
I'm tempted to have a dedicated pipeline for each example, instead of adding build time to the main one. Wdyt? If that's the case, should the checks be mandatory for the review?